tests/js/designer/diagram/events/timereventdefinition.test.js

```diff
@@ -12,17 +12,21 @@ describe("Elements", () => {
     beforeEach(() => {
         graph = new joint.dia.Graph
         paper = {}
-        ev = new Elements["timereventdefinition"]({
+        ev = new Elements["intermediatethrowevent"]({
```

Comment:
Why the file's test is named timereventdefinition.test.js but here it seems that we are testing an intermediatethrowevent?